ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-pim-sparse-mode.md

```diff
@@ -105,6 +107,8 @@ interface Management1
 
 ##### IP Sparse Mode VRFs
 
+BFD enabled: False
```

Comment:
This would not be a common setting, but a setting per VRF, so if we want to document it, it should be a new column in the table below, or as a separate table.

Reply:
Good catch. Updated as a separate table, I think it's more clear than having it on the same table as the RP info...